src/lib/src/hal/hal.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { HttpModule } from '@angular/http';
+
+import { HalService } from './hal.service';
+import './doc/halobservable';
```

Comment:
This line is what actually modifies `Observable.prototype` (since we can't actually extend Observable).  So including this module anywhere will guarantee HalObservable gets setup correctly.